Mathlib/Topology/Bornology/Absorbs.lean

```diff
@@ -133,58 +133,70 @@ protected lemma zero [Zero E] [SMulZeroClass M E] {s : Set E} (hs : 0 ∈ s) : A
 
 end AddZero
 
+end Absorbs
+
 section GroupWithZero
 
 variable {G₀ α : Type*} [GroupWithZero G₀] [Bornology G₀] [MulAction G₀ α]
   {s t u : Set α} {S : Set (Set α)}
 
 @[simp]
-protected lemma univ : Absorbs G₀ univ s :=
+protected lemma Absorbs.univ : Absorbs G₀ univ s :=
   (eventually_ne_cobounded 0).mono fun a ha ↦ by rw [smul_set_univ₀ ha]; apply subset_univ
 
-lemma _root_.absorbs_iff_eventually_cobounded_mapsTo :
+lemma absorbs_iff_eventually_cobounded_mapsTo :
     Absorbs G₀ s t ↔ ∀ᶠ c in cobounded G₀, MapsTo (c⁻¹ • ·) t s :=
   eventually_congr <| (eventually_ne_cobounded 0).mono fun c hc ↦ by
     rw [← preimage_smul_inv₀ hc]; rfl
 
 alias ⟨eventually_cobounded_mapsTo, _⟩ := absorbs_iff_eventually_cobounded_mapsTo
 
-lemma _root_.Set.Finite.absorbs_sInter (hS : S.Finite) :
-    Absorbs G₀ (⋂₀ S) t ↔ ∀ s ∈ S, Absorbs G₀ s t := by
-  simp only [absorbs_iff_eventually_cobounded_mapsTo, mapsTo_sInter, hS.eventually_all]
+@[simp]
+lemma absorbs_inter : Absorbs G₀ (s ∩ t) u ↔ Absorbs G₀ s u ∧ Absorbs G₀ t u := by
+  simp only [absorbs_iff_eventually_cobounded_mapsTo, mapsTo_inter, eventually_and]
+
+protected lemma Absorbs.inter (hs : Absorbs G₀ s u) (ht : Absorbs G₀ t u) : Absorbs G₀ (s ∩ t) u :=
+  absorbs_inter.2 ⟨hs, ht⟩
 
-protected alias ⟨_, sInter⟩ := Set.Finite.absorbs_sInter
+variable (G₀ u) in
+/-- The filter of sets that absorb `u`. -/
+def Filter.absorbing : Filter α where
```

Comment:
Is this worth a mention in the module docstring and/or the docstring for `Absorbs`?